src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -1107,7 +1107,9 @@ namespace detail
         adios2::Variable< T > var = m_impl->verifyDataset< T >(
             bp.param.offset, bp.param.extent, IO, bp.name );
 
-        engine.Put( var, ptr );
+    // using sync mode has the same effect as deferred and less memory consumption
+    // at of now.
```

Comment:
Ideally this file change goes in a separate pull request, since it's unrelated to the tests and would be better documented that way.

Cosmetic fix:
```suggestion
        // using sync mode has the same effect as deferred and less memory consumption
        // as of ADIOS 2.6.0.
```